Sources/DeepLinking/Route+Component.swift

```diff
@@ -53,7 +53,7 @@ public extension Route {
 
         // MARK: Key
 
-        public enum Key {
+        public enum Key: Hashable, Equatable {
```

Comment:
I believe `Hashable` inherits from `Equatable`. I think you should be able to just conform to `Hashable` here, which gives you `Equatable` conformance for free.

Reply:
Yep, that's right! Thanks! 🙂